.github/workflows/macos.yml

```diff
@@ -101,6 +101,9 @@ jobs:
         env:
           PROJ_LIB: ${{ env.CMAKE_BUILD_DIR }}/vcpkg_installed/x64-osx/share/proj/data
           QFIELD_SYSTEM_SHARED_DATA_PATH: ${{ env.CMAKE_BUILD_DIR }}/vcpkg_installed/x64-osx/share
+          QFIELD_FONT_TTF: ${{ github.workspace }}/test/testdata/FiraSansCondensed-Regular.ttf
```

Comment:
This can all be hardcoded in the tests. It should also be the same when run outside the ci infra.

Reply:
💡